src/OpenTelemetry.Contrib.Instrumentation.AWS/OpenTelemetry.Contrib.Instrumentation.AWS.csproj

```diff
@@ -1,18 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
I think as long as we are not using any features from the new versions, we should not upgrade yet.

Reply:
OpenTelemetry 1.3.0 is no longer compatible with .NET Framework 4.5.2. The latest compatible version is 1.1.0.

Ref: https://www.nuget.org/packages/OpenTelemetry/1.3.0#supportedframeworks-body-tab

Reply:
OK, but is there any advantage from updating OTel from 1.1 yet? Any new features used?

Reply:
I do not see any direct feature, but I see some benefits:
 - promote newer OTel release. In case of any issues, the user will be asked to verify how it is working with the newer version.
 - Drop support for not supported .NET (Framework) versions https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework
  
 BTW my previous comment should suggest that we take both changes or any of them.